Dockerfile

```diff
@@ -1,19 +1,33 @@
 FROM ruby:2.7.1
+
 RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
     && echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
-RUN apt-get update -qq && apt-get install -y nodejs postgresql-client yarn
+
+RUN apt-get update -qq && \
+    apt-get install -y --no-install-recommends \
+      nodejs \
+      postgresql-client \
+      yarn
+
 RUN mkdir /myapp
 WORKDIR /myapp
+
 COPY Gemfile /myapp
 COPY Gemfile.lock /myapp
+
 RUN bundle install
+
+COPY package.json /myapp
+COPY yarn.lock /myapp
+
+RUN yarn install
+
 COPY . /myapp
```

Comment:
We still need this instruction.
If not, deploying to heroku would fail.

Reply:
Why is the deploy fail? In Heroku, is `Dockerfile` is used to deploy?

In case of Heroku, we need to update `Procfile` because the file is about the same as `Dockerfile` and `docker-compose.yml` in my understanding.

Reply:
I see I misunderstood.
But let me leave it for now because this instruction in the current Dockerfile has no pain point.

Reply:
OK. Then, could you please create new issue for improvement of `Procfile`?